tests/test_adaptor_increase_maxorder.cc

```diff
@@ -182,23 +182,23 @@ namespace rascal {
         auto neighbour_position = pair.get_position();
         auto diff_pos_pair = (neighbour_position - atom_position).norm();
         BOOST_CHECK_CLOSE(diff_pos_pair, 1., TOLERANCE);
+      }
 
-        for (auto triplet : pair) {
-          if (verbose) {
-            std::cout << "triplet " << atom.back() << " " << pair.back() << " "
-                      << triplet.back() << std::endl;
-          }
-          auto neighbour_of_neighbour_atom_tag =
-              triplet.get_internal_neighbour_atom_tag();
-          auto neighbour_of_neighbour_type = triplet.get_atom_type();
-          BOOST_CHECK_EQUAL(
-              neighbour_of_neighbour_type,
-              SM3->get_atom_type(neighbour_of_neighbour_atom_tag));
-
-          auto triplet_position = triplet.get_position();
-          auto diff_pos_triplet = (triplet_position - atom_position).norm();
-          BOOST_CHECK_CLOSE(diff_pos_triplet, 1., TOLERANCE);
+      for (auto triplet : atom.triplets()) {
+        if (verbose) {
+          auto tags = triplet.get_atom_tag_list();
```

Comment:
Do you want to add a test for lexicographical sorting of the tags here?

Reply:
yes